socket/packet/server_list_response.go

```diff
@@ -0,0 +1,60 @@
+package packet
+
+import (
+	"github.com/sandertv/gophertunnel/minecraft/protocol"
+)
+
+// ServerListResponse is sent by the proxy in response to ServerListRequest. It sends list of all
+// the servers connected to the proxy.
```

Comment:
Same thing here with the character limit

Reply:
That's 94 chars